### PR TITLE
Update dev_setup.py script

### PR DESCRIPTION
**Pull request by troydai**
1. Make it executable
2. Update shebang
3. Colorize the output and improve the readability

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@troydai__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (trdai). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: scripts/dev_setup.py [hunk 1]
```diff
@@ -1,31 +1,41 @@
...
+#   green \033[92m
+#   blue  \033[94m
+#   end   \033[0m
+
 def exec_command(command):
     try:
-        print('Executing: ' + command)
+        print('\033[92mExecuting: ' + command + '\033[0m')
         check_call(command.split(), cwd=root_dir)
+        print('\n')
```
Comment (derekbekoe) [inline scripts/dev_setup.py]: Use `print()`.
`print('\n')` is the same as `print()`:
see https://docs.python.org/3/library/functions.html#print

File: scripts/dev_setup.py [hunk 2]
```diff
@@ -1,31 +1,41 @@
...
 
+# color code used in the print statement:
+#   green \033[92m
+#   blue  \033[94m
+#   end   \033[0m
+
 def exec_command(command):
     try:
-        print('Executing: ' + command)
+        print('\033[92mExecuting: ' + command + '\033[0m')
```
Comment (derekbekoe) [inline scripts/dev_setup.py]: The color codes may not work on Windows cmd or other terminal that don't support colors.

Comment (troydai) [inline scripts/dev_setup.py]: ok. i'll remove the color codes then.

**Comment by troydai** [comment]
Removed the color code and squash the code. The commit is rebase on latest master, it is ready to merge after CI passed.

**Pull request merged**